app/main.py

```diff
@@ -1,6 +1,16 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def inner(*args, **kwargs) -> Any:
```

Comment:
if you also want to add kwargs here, you should handle them in another way. You can store args and kwargs as a dictionary key or even simply delete kwargs in this task) Up to you 😄